src/app/component/eco-news/components/create-news/create-news.component.ts

```diff
@@ -91,9 +91,9 @@ export class CreateNewsComponent implements OnInit {
 
   public onSourceChange(): void {
     this.createNewsForm.get('source').valueChanges.subscribe(source => {
-      source.startsWith('http://') ||
-      source.startsWith('https://') ||
-      source.length === 0 ? this.isLink = false : this.isLink = true;
+      this.isLink = source.startsWith('http://') ||
+                    source.startsWith('https://') ||
+                    !!source.length;
```

Comment:
The style is a lot better. But look here:
![image](https://user-images.githubusercontent.com/6242270/89629419-7528cb80-d8a6-11ea-9237-9b863d1449a7.png)
Any string will return `true`. An empty string is exception. Actually it's not after your changing - the same effect will be if we keep previous code.
So, I'd like to ask you to investigate it - what is this checking? What is reason? How will it work if we remove this source.length? In result - how this code can be changed to became easier and clearer.

Reply:
fixed